cli/src/merge_tools/external.rs

```diff
@@ -276,6 +276,15 @@ fn run_mergetool_external_single_file(
         return Err(ConflictResolveError::EmptyOrUnchanged);
     }
 
+    if exit_status_implies_conflict {
+        tracing::info!(
+            "Parsing conflicts in output file due to tool exiting with code {:?}",
+            // The code will always be Some(...), but we never want to panic here if logic changes
+            exit_status.code().unwrap_or(-9999)
+        );
+    } else if editor.merge_tool_edits_conflict_markers {
+        tracing::info!("Parsing conflicts in output file due to tool config");
+    }
```

Comment:
nit: better to just dump raw values? `tracing::debug!(?exit_status, edits_conflict_marker = ?editor.merge_tool_edits_conflict_marker, ...)`

I think tracing code should be kept simple.

Reply:
Aside: I'm also wondering about adding a debugging mode where `jj` pauses after invoking the merge tool and gives the option not to delete the temporary dir. I'm running into naming problems, though, since `--debug` is already taking. I was thinking `--config ui.debug-merge-tools=true`, but again, the `debug` here can be confused with `--debug`.

If that happens at some point, we could maybe stop using tracing for user-facing debug messages.

Reply:
Hmm, then Display works better? `%exit_status`. I also think it's better to print `%exit_status` unconditionally. When debugging merge-tools behavior, you'll probably want to see it no matter if conflicts are implied or not.

> debugging mode where jj pauses after invoking the merge tool and gives the option not to delete the temporary dir

Adding env var might be okay.

Reply:
`%exist_status` works, thanks! How about this?